src/styles/main.scss

```diff
@@ -1,14 +1,4 @@
-html {
-  font-family: Roboto, sans-serif;
-}
-
-body {
-  margin: 0;
-}
-
-* {
-  box-sizing: border-box;
-}
-
-@import "_header.scss";
-@import "_catalog.scss";
+@import "./utils/reset";
+@import "./utils/variables";
+@import "./bloks/header";
+@import "./bloks/catalog";
```

Comment:
The import path should be './bloks/_catalog.scss' to keep consistent with the task description.